modules/ROOT/pages/con_pipeline-cluster-resource.adoc

```diff
@@ -0,0 +1,48 @@
+[]
+= Cluster Resource
+
+
+A Cluster Resource represents a Kubernetes cluster other than the current cluster Pipelines is running on. A common use case for this resource is to deploy your application or function on different clusters.
+
+The resource will use the provided parameters to create a
+[kubeconfig](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/)
+file that can be used by other steps in the pipeline task to access the target
+cluster. The kubeconfig will be placed in `/workspace/<your-cluster-name>/kubeconfig` on your Task container
```

Comment:
- The kubeconfig--> "The kubeconfig file". If you agree to use backticks,  "The `kubeconfig` file".

- No future tense: "The `kubeconfig` file is placed on your Task container. 
-Period after the sentence.